sql_generators/glean_usage/event_monitoring_live.py

```diff
@@ -39,7 +39,17 @@ def __init__(self) -> None:
         self.custom_render_kwargs = {}
         self.base_table_name = "events_v1"
 
-    def _get_tables_with_events(self, v1_name: str) -> Set[str]:
+    def _get_prod_datasets_with_event(self) -> List[str]:
+        """Get glean datasets with an events table in generated schemas."""
+        return [
+            s.bq_dataset_family
+            for s in get_stable_table_schemas()
+            if s.schema_id == "moz://mozilla.org/schemas/glean/ping/1"
+            and s.bq_table == "events_v1"
+        ]
```

Comment:
This was a bit of a hacky way to get all relevant datasets, since (almost?) all of them do have `event` pings.